js/packages/redux/reducers/ui.reducer.ts

```diff
@@ -218,54 +135,41 @@ const slice = createSlice({
 		setDaemonAddress(state: UiState, { payload: { value } }: PayloadAction<{ value: string }>) {
 			state.daemonAddress = value
 		},
-		setStateOpeningListingEvents(
+		setStateClients(
 			state: UiState,
 			{
 				payload,
 			}: PayloadAction<{
 				messengerClient?: ServiceClientType<beapi.messenger.MessengerService> | null
 				protocolClient?: ServiceClientType<beapi.protocol.ProtocolService> | null
-				clearClients?: (() => Promise<void>) | (() => void) | null
 			}>,
 		) {
-			state.client = payload.messengerClient || state.client
+			state.messengerClient = payload.messengerClient || state.messengerClient
 			state.protocolClient = payload.protocolClient || state.protocolClient
-			state.clearClients = payload.clearClients || state.clearClients
-			changeAppState(state, MESSENGER_APP_STATE.OPENING_LISTING_EVENTS)
 		},
-		setStateClosed: setStateClosedFn,
 		setNextAccount(state: UiState, { payload }: PayloadAction<string | null | undefined>) {
-			if (payload === null || payload === undefined || !state.embedded) {
-				return
-			}
-			state.nextSelectedAccount = payload
-			setStateClosedFn(state)
-		},
-		setStateOpening: setStateOpeningFn,
-		setStateOpeningClients(state: UiState) {
-			state.appState = MESSENGER_APP_STATE.OPENING_WAITING_FOR_CLIENTS
+			Object.keys(initialState).map(k => {
+				if (['accounts'].indexOf(k) === -1) {
+					//@ts-ignore
+					state[k] = initialState[k as keyof UiState]
+				}
+			})
```

Comment:
```suggestion
			state.accounts = initialState.accounts
```

Reply:
nope it's for not reset accounts

Reply:
sorry I read the condition as reverse, I get it now, maybe just do `if (k !== 'accounts')`